### PR TITLE
Don't allow serialization if dust amounts are detected

### PR DESCRIPTION
**Pull request by eordano**

### CODE REVIEW

**Comment by maraoz** [comment]
travis failed

**Comment by yemel** [comment]
LGTM

File: test/transaction/deserialize.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ describe('Transaction deserialization', function() {
     vectors_valid.forEach(function(vector) {
       if (vector.length > 1) {
         var hexa = vector[1];
-        Transaction(hexa).serialize().should.equal(hexa);
+        Transaction(hexa).serialize(true).should.equal(hexa);
```
Comment (maraoz) [inline test/transaction/deserialize.js]: does this boolean indicate unsafe serialization?

**Comment by maraoz** [comment]
ACK, never mind the question

**Pull request merged**